dulwich/protocol.py

```diff
@@ -30,6 +30,10 @@
 
 TCP_GIT_PORT = 9418
 
+GIT_PROTOCOL_VERSIONS = [0, 1, 2]
```

Comment:
Is there actually a version 1? Can you add a comment about these versions.

Reply:
Yes, I am adding comments in the next version.

They will appear in an earlier commit which adds this list.
I have rewritten history a bit, also to get rid of the 'ruff format' commit in the middle.